.github/workflows/dev-cluster-check.yml

```
@@ -0,0 +1,34 @@
+name: Check development cluster
+on:
+  schedule:
+    - cron: '* */3 * * *'
+jobs:
+  run-integration-tests:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Free disk space
+        uses: ./.github/actions/free-disk-space
+      - name: Java and Maven Setup
+        uses: ./.github/actions/java-maven-setup
+        with:
+          cache-key-prefix: ${{ runner.os }}
+      - name: Compile sandbox repo
+        uses: ./.github/actions/maven
+        with:
+          maven-command: clean verify
```

```suggestion
          maven-command: clean verify -DskipTests
```
To speed up the execution.